spec/controllers/static_pages_controller_spec.rb

```diff
@@ -28,4 +28,10 @@
       expect(privacy_policy_path).to eq "/privacy-policy"
     end
   end
+  describe "cookies" do
+    it "should respond to /cookies" do
+      expect({:get => "/cookies"}).to route_to({:controller => "static_pages", :action => "cookies"})
+      expect(cookies_path).to eq "/cookies"
+    end
+  end  
```

Comment:
This is a routing spec and should live in `spec/routing` but since you're just adding to the existing ones, I'll let it slide :wink: